realhf/impl/model/backend/pipe_runner.py

```diff
@@ -371,6 +375,9 @@ def _exec_forward_pass(
         micro_batch_id: int,
         step_id: int,
     ):
+        cuda_graph_name = f"decoding_{micro_batch_id}"
```

Comment:
Use the same cuda graph for all micro batches?

Reply:
As discussed, we should remain separate KV caches for different micro batches, so we have to reserve individual CUDA graphs.